src/Helper/ChunkHelpers.php

```diff
@@ -47,19 +49,29 @@ public function mergeChunks(StorageConfig $config, array $chunks, string $target
     }
 
     /**
+     * Delete a directory with the given name from the chunk directory.
+     *
+     * @param \LaraCrafts\ChunkUploader\StorageConfig $config
+     * @param string $uuid
+     */
+    public function deleteChunkDirectory(StorageConfig $config, string $uuid): void
+    {
+        $directory = $config->getChunkDirectory() . '/' . $uuid;
```

Comment:
At first this could seem as a "useless" variable because it is only used once. However, the variable name it self contains information about the value it has.